databricks/koalas/groupby.py

```diff
@@ -2018,8 +2018,8 @@ def __init__(self, kser: Series, by: List[Series], as_index: bool = True):
         #   `SeriesGroupBy` creates another DataFrame and
         #   internal IDs of the columns become different. Maybe we should refactor the whole
         #   class in the future.
-        self._groupkeys_scols = [F.col(name_like_string(s.name)) for s in self._groupkeys]
-        self._agg_columns_scols = [F.col(name_like_string(s.name)) for s in self._agg_columns]
+        self._groupkeys_scols = [F.col(s._internal.data_columns[0]) for s in self._groupkeys]
+        self._agg_columns_scols = [F.col(s._internal.data_columns[0]) for s in self._agg_columns]
```

Comment:
I think now we can enable the tests back (https://github.com/databricks/koalas/pull/1224#discussion_r370972255). I will make a small followup.